Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -795,12 +787,17 @@ open class NavigationMapView: UIView {
             case .puck3D(configuration: let configuration):
                 mapView.location.options.puckType = .puck3D(configuration)
             case .none:
-                mapView.location.options.puckType = .puck2D(emptyPuckConfiguration)
+                mapView.location.options.puckType = nil
             }
             mapView.location.options.puckBearingSource = .course
         }
     }
     
+    func setupLocationManager() {
+        locationManager.desiredAccuracy = kCLLocationAccuracyThreeKilometers
```

Comment:
Why do you need this accuracy parameter to be set? I don't see where this location manager is started.

Reply:
Not necessary, will be removed.

Reply:
To listen to `CLLocationManagerDelegate` updates there is no need to start `CLLocationManager`.